### ISSUE TITLE
It's easy to inadvertantly submit duplicate POST requests in quick succession

### ISSUE THREAD

**Issue by amusselm** [amusselm/rvgs_gamelist#29]
A user can accidentally click the "submit" button in some forms more than once in rapid succession, leading to duplicate enteries. 

The system should have a means of de-dupulicating these. Or preventing this from happening in the first place.

**Comment by snoogans775** [comment]
in PHP there are functions for removing duplicate entries in an array. This seems to be an equivalent in Python: [(http://stackoverflow.com/questions/479897/how-to-remove-duplicates-from-python-list-and-keep-order)]

**Comment by amusselm** [comment]
Yeah, a regex search of the database might be a better answer than trying
to "debounce" user input. Users might be unaware a game exists in the
database, so a "did you mean" screen if a similar title exists in the
database might be worthwhile.
